commands/expand.js

```diff
@@ -0,0 +1,25 @@
+const expand = require('expand-string-professionally')
+
+module.exports = {
+    name: 'expand',
+    description: 'For when a string must be expanded',
+    execute(message, args, newEmbed) {
+        if (args.length === 0)
+            return message.reply(`You didn't give me a string to expand, asshole`);
```

Comment:
Please change the error reply to conform more with the code already present.

Reply:
Asshole, just merge it